pkg/operator/operator_test.go

```diff
@@ -43,40 +43,55 @@ func TestEnsureCertsExplicit(t *testing.T) {
 	defer os.RemoveAll(dir)
 
 	for _, tc := range []struct {
-		desc       string
-		opts       Options
-		expectCert string
-		expectKey  string
-		expectErr  bool
+		desc         string
+		opts         Options
+		expectCert   string
+		expectKey    string
+		expectCaCert string
+		expectErr    bool
 	}{
 		{
-			desc:       "input key/cert",
-			opts:       Options{TLSKey: "a2V5", TLSCert: "Y2VydA==", OperatorNamespace: "test-ns"},
-			expectCert: "cert",
-			expectKey:  "key",
-			expectErr:  false,
+			desc:         "input key/cert/ca",
+			opts:         Options{TLSKey: "a2V5", TLSCert: "Y2VydA==", CACert: "Y2FjZXJ0", OperatorNamespace: "test-ns"},
+			expectCert:   "cert",
+			expectKey:    "key",
+			expectCaCert: "cacert",
+			expectErr:    false,
+		},
+		{
+			desc:         "cert/key and no CA",
+			opts:         Options{TLSKey: "a2V5", TLSCert: "Y2VydA==", CACert: "Y2VydA==", OperatorNamespace: "test-ns"},
```

Comment:
This doesn't seem correct. The test case says no CA is set, but the options do have CACert have it set – just to the same value as TLSCert.